crates/matrix-sdk-ui/src/timeline/event_item/content/message.rs

```diff
@@ -279,7 +286,7 @@ impl RepliedToEvent {
         }
     }
 
-    pub(in crate::timeline) async fn try_from_timeline_event<P: RoomDataProvider>(
+    pub async fn try_from_timeline_event<P: RoomDataProvider>(
```

Comment:
Are we sure about this change?
If it goes public, it must have a documentation at least.

Reply:
Made this function not public, and instead created a wrapper function around it that hides the generic usage (since RoomDataProvider would be more private than the actual function in that case) for hide this one. Also wrote a small piece of documentation for it.